Fitter/TauES_ID/harvestDatacards_TES_idSF_MCStat.py

```diff
@@ -316,11 +332,12 @@ def main(args):
   argv = sys.argv
   description = '''This script makes datacards with CombineHarvester.'''
   parser = ArgumentParser(prog="harvesterDatacards_TES",description=description,epilog="Succes!")
-  parser.add_argument('-y', '--year', dest='year', choices=['2016','2017','2018','UL2016_preVFP','UL2016_postVFP','UL2017','UL2018','UL2018_v10','2022_postEE','2022_preEE'], type=str, default=2018, action='store', help="select year")
+  parser.add_argument('-y', '--year', dest='year', type=str, default=2018, action='store', help="select year")
   parser.add_argument('-c', '--config', dest='config', type=str, default='TauES/config/defaultFitSetupTES_mutau.yml', action='store', help="set config file containing sample & fit setup")
   parser.add_argument('-e', '--extra-tag', dest='extratag', type=str, default="", action='store', metavar='TAG', help="extra tag for output files")
   parser.add_argument('-M', '--multiDimFit', dest='multiDimFit', default=False, action='store_true', help="assume multidimensional fit with a POI for each DM")
   parser.add_argument('-v', '--verbose', dest='verbose', default=False, action='store_true', help="set verbose")
+  parser.add_argument('-i', '--input_dir', dest='input_dir', type=str, help='input_dir for datacard root file')
```

Comment:
Also for this script (as the createinputs) it would be nice to put an example of how to run it (with all the arguments used) in the README